integration-tests/src/tests/client/sandbox.rs

```diff
@@ -109,3 +118,40 @@ fn test_patch_account() {
     let test1_after = env.query_account("test1".parse().unwrap());
     assert_eq!(test1_after.amount, 10);
 }
+
+#[test]
+#[cfg(feature = "sandbox")]
```

Comment:
By the way, `#[cfg(feature = "sandbox")]` isn’t necessary. Neither here nor anywhere else in the file. The file is conditionally included in `mod.rs` and won’t be compiled if `sandbox` feature is not enabled.